### DISCUSSION TITLE
Use from WebAssembly runtime

### DISCUSSION THREAD

**Discussion by h3x4d3c1m4l** [dlemstra/magick-wasm#34] [Q&A]
Hi all,

Is it possible to use ImageMagick WASM without JS glue code?
This could make it possible to use it in languages/frameworks that don't have any ImageMagick bindings like Dart/Flutter.
https://pub.dev/documentation/wasm/latest/

**Comment by dlemstra** [comment]
It might be possible? I think I can create a `.wasm` file in the build of [Magick.Native](https://github.com/dlemstra/Magick.Native). Will require some experimenting from your or my side.

**Comment by h3x4d3c1m4l** [comment]
Sounds fun, would love to test. I'm curious about the performance, and I have an actual usecase for it.

I would like to create a simple open source Dart based webservice for generating test images.
